index.html

```diff
@@ -1,19 +1,103 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma Logo" class="nav-logo" />
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item-meet-karma">
+            <a class="nav-link-meet-karma" href="#">Meet Karma</a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="store.html">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <main>
+      <section class="background-image">
+        <div class="karma">
+          <h1>Introducing Karma</h1>
+          <p>Bring WiFi with you, everywhere you go.</p>
+          <button class="button">Learn More</button>
+        </div>
+      </section>
 
-</body>
+      <section class="details">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="article-container">
+          <article>
+            <img src="img/icon-devices.svg" alt="" class="" />
+            <p>Internet for all devices</p>
+          </article>
+          <article>
+            <img src="img/icon-coffee.svg" alt="" class="" />
+            <p>Boost your productivity</p>
+          </article>
+          <article>
+            <img src="img/icon-refill.svg" alt="" class="" />
+            <p>Pay as You Go</p>
+          </article>
+        </div>
+      </section>
+
+      <section class="get-karma">
+        <img src="additional/feature-img.png" alt="" class="feature-img" />
+        <div class="rect-get-karma">
+          <p>
+            <span style="color: #e0633a">"</span>Wherever I am, I just don't
```

Comment:
Do not use explicit styles / colors in your HTML markup - these things should be in CSS. Otherwise it's difficult to modify - you need to update both files if you want to change your accent color